source/max-staleness/max-staleness-tests.rst

```diff
@@ -87,6 +88,8 @@ but less than 10 seconds greater.
 while still testing that the driver parses lastWriteDate
 and converts it to an appropriate unit.)
 
+.. [1] This ensures the next insert occurs at least one second after server selection (another write may have occurred from the noop writer just before server selection if mongod was configured with periodicNoopIntervalSecs=1).
+
```

Comment:
Can you wrap this line at 80 chars?

Reply:
Done.

Reply:
Sorry, the newlines need to be indented: https://docutils.sourceforge.io/docs/user/rst/quickref.html#footnotes

```
.. [1] This...
   selection (another...
   server selection...
```

Reply:
One day I will learn how to speak rst...